cirq/experiments/generate_google_v2_random_circuit.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Callable, Iterable, TypeVar
+
+import cirq
+from cirq.circuits import InsertStrategy
+from cirq.devices.grid_qubit import GridQubit
+from cirq.ops import common_gates
```

Comment:
`from cirq import ops`

When in a neighboring module, import at the module level instead of at the file level.